roles/common/tasks/kernel-tuning.yml

```diff
@@ -41,6 +45,7 @@
   lineinfile: dest=/etc/default/grub
               regexp="^GRUB_CMDLINE_LINUX="
               line="GRUB_CMDLINE_LINUX=\"consoleblank=0 nmi_watchdog=1 {{ serial_console_cmdline|default('') }} console=tty0 biosdevname=0 {{- ' maxcpus=%s' % cpulimit if cpulimit is defined else '' }} {{- 'smt-enabled=off' if ansible_architecture == 'ppc64le' else '' }} \""
+  when: grub_defaults.stat.exists == True | bool
```

Comment:
This should be
`when: grub_defaults.stat.exists`

The stat module returns a boolean.